.github/workflows/docker.yml

```diff
@@ -1,37 +0,0 @@
-name: Build and Push Image samply/lens
```

Comment:
For automatic testing, I would prefer to run docker in the ci. I would just run the docker build command and try to serve the demo. This is would replace testing, but would show if there is an error while building